subprojects/docs/src/main/resources/header.html

```diff
@@ -71,16 +71,10 @@
     <!-- Primary Navigation -->
     <nav class="docs-navigation">
         <ul>
-            <li class="main-link"><a href="/userguide/userguide.html">User Manual</a>
-                <ul class="manual-formats">
-                    <li><a href="/userguide/userguide_single.html">1 page</a></li>
-                    <li><a href="/userguide/userguide.pdf">PDF</a></li>
-                </ul>
-            </li>
-            
-            <li class="main-link"><a href="/release-notes.html">Release Notes</a></li>
-            <li class="main-link"><a href="/userguide/installation.html">Installing Gradle</a></li>
-            <li class="main-link" id="tutorials"><a href="https://guides.gradle.org/">Tutorials</a></li>
+            <li><a href="/userguide/userguide.html">User Manual Home</a></li>
```

Comment:
Sticking with "User Manual Home"? I don't feel particularly strongly about this one either way. Just want to check that this was a deliberate roll back.